[2024-01-29T09:22:00.453+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manage_data_lake_dag.spark_job_formatted manual__2024-01-29T09:21:50.369923+00:00 [queued]>
[2024-01-29T09:22:00.463+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manage_data_lake_dag.spark_job_formatted manual__2024-01-29T09:21:50.369923+00:00 [queued]>
[2024-01-29T09:22:00.463+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-29T09:22:00.475+0000] {taskinstance.py:2191} INFO - Executing <Task(SparkSubmitOperator): spark_job_formatted> on 2024-01-29 09:21:50.369923+00:00
[2024-01-29T09:22:00.479+0000] {standard_task_runner.py:60} INFO - Started process 80841 to run task
[2024-01-29T09:22:00.484+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'manage_data_lake_dag', 'spark_job_formatted', 'manual__2024-01-29T09:21:50.369923+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/data_lake_dag.py', '--cfg-path', '/tmp/tmprllg8rzl']
[2024-01-29T09:22:00.486+0000] {standard_task_runner.py:88} INFO - Job 75: Subtask spark_job_formatted
[2024-01-29T09:22:00.521+0000] {task_command.py:423} INFO - Running <TaskInstance: manage_data_lake_dag.spark_job_formatted manual__2024-01-29T09:21:50.369923+00:00 [running]> on host instance-1.europe-west9-a.c.data-lake-project-409321.internal
[2024-01-29T09:22:00.580+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='juniortemgoua0' AIRFLOW_CTX_DAG_ID='manage_data_lake_dag' AIRFLOW_CTX_TASK_ID='spark_job_formatted' AIRFLOW_CTX_EXECUTION_DATE='2024-01-29T09:21:50.369923+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-29T09:21:50.369923+00:00'
[2024-01-29T09:22:00.586+0000] {base.py:83} INFO - Using connection ID 'spark_default' for task execution.
[2024-01-29T09:22:00.587+0000] {spark_submit.py:401} INFO - Spark-Submit cmd: spark-submit --master yarn --name arrow-spark --class RawToFormatted --queue root.default --deploy-mode client /home/juniortemgoua0/DataLake/jobs/processes/scala/spark_process/target/scala-2.12/spark_job_2.12-0.1.0.jar gs://data-lake-buck
[2024-01-29T09:22:03.072+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:03 INFO SparkContext: Running Spark version 3.5.0
[2024-01-29T09:22:03.076+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:03 INFO SparkContext: OS info Linux, 5.15.0-1049-gcp, amd64
[2024-01-29T09:22:03.083+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:03 INFO SparkContext: Java version 17.0.9
[2024-01-29T09:22:03.235+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:03 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-01-29T09:22:03.394+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:03 INFO ResourceUtils: ==============================================================
[2024-01-29T09:22:03.396+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:03 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-01-29T09:22:03.397+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:03 INFO ResourceUtils: ==============================================================
[2024-01-29T09:22:03.398+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:03 INFO SparkContext: Submitted application: RawToFormatted
[2024-01-29T09:22:03.448+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:03 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-01-29T09:22:03.463+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:03 INFO ResourceProfile: Limiting resource is cpus at 1 tasks per executor
[2024-01-29T09:22:03.466+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:03 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-01-29T09:22:03.546+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:03 INFO SecurityManager: Changing view acls to: juniortemgoua0
[2024-01-29T09:22:03.547+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:03 INFO SecurityManager: Changing modify acls to: juniortemgoua0
[2024-01-29T09:22:03.548+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:03 INFO SecurityManager: Changing view acls groups to:
[2024-01-29T09:22:03.549+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:03 INFO SecurityManager: Changing modify acls groups to:
[2024-01-29T09:22:03.550+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:03 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: juniortemgoua0; groups with view permissions: EMPTY; users with modify permissions: juniortemgoua0; groups with modify permissions: EMPTY
[2024-01-29T09:22:03.903+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:03 INFO Utils: Successfully started service 'sparkDriver' on port 35085.
[2024-01-29T09:22:03.942+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:03 INFO SparkEnv: Registering MapOutputTracker
[2024-01-29T09:22:03.996+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:03 INFO SparkEnv: Registering BlockManagerMaster
[2024-01-29T09:22:04.020+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:04 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-01-29T09:22:04.020+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:04 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-01-29T09:22:04.068+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:04 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-01-29T09:22:04.088+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:04 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-32ca7e88-3821-47bb-92bd-214faf27148d
[2024-01-29T09:22:04.106+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:04 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-01-29T09:22:04.160+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:04 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-01-29T09:22:04.351+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:04 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-01-29T09:22:04.445+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:04 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-01-29T09:22:04.508+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:04 INFO SparkContext: Added JAR file:/home/juniortemgoua0/DataLake/jobs/processes/scala/spark_process/target/scala-2.12/spark_job_2.12-0.1.0.jar at spark://instance-1.europe-west9-a.c.data-lake-project-409321.internal:35085/jars/spark_job_2.12-0.1.0.jar with timestamp 1706520123055
[2024-01-29T09:22:04.782+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:04 INFO DefaultNoHARMFailoverProxyProvider: Connecting to ResourceManager at /0.0.0.0:8032
[2024-01-29T09:22:05.988+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:05 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:06.990+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:06 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:07.991+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:07 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:08.992+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:08 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:09.993+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:09 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:10.995+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:10 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:11.995+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:11 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:12.996+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:13.997+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:14.998+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:16.012+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:17.013+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:18.013+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:18 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:19.014+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:19 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:20.015+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:20 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:21.016+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:21 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:22.017+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:22 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:23.017+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:23 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:24.018+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:24 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:25.019+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:25 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:22:25.023+0000] {spark_submit.py:571} INFO - 24/01/29 09:22:25 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 1 failover attempts. Trying to failover after sleeping for 38066ms.
[2024-01-29T09:23:04.091+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:04 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:05.092+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:05 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:06.093+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:06 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:07.094+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:07 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:08.096+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:08 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:09.096+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:09 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:10.097+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:10 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:11.098+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:11 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:12.099+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:13.100+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:13.105+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:13 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 2 failover attempts. Trying to failover after sleeping for 31329ms.
[2024-01-29T09:23:45.436+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:45 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:46.437+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:46 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:47.438+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:47 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:48.439+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:48 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:49.440+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:49 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:50.442+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:50 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:51.443+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:51 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:52.443+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:52 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:53.444+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:53 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:54.445+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:54 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:23:54.449+0000] {spark_submit.py:571} INFO - 24/01/29 09:23:54 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 3 failover attempts. Trying to failover after sleeping for 16663ms.
[2024-01-29T09:24:12.114+0000] {spark_submit.py:571} INFO - 24/01/29 09:24:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:24:13.115+0000] {spark_submit.py:571} INFO - 24/01/29 09:24:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:24:14.115+0000] {spark_submit.py:571} INFO - 24/01/29 09:24:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:24:15.116+0000] {spark_submit.py:571} INFO - 24/01/29 09:24:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:24:16.117+0000] {spark_submit.py:571} INFO - 24/01/29 09:24:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:24:17.118+0000] {spark_submit.py:571} INFO - 24/01/29 09:24:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:24:18.119+0000] {spark_submit.py:571} INFO - 24/01/29 09:24:18 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:24:19.120+0000] {spark_submit.py:571} INFO - 24/01/29 09:24:19 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:24:20.121+0000] {spark_submit.py:571} INFO - 24/01/29 09:24:20 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:24:21.122+0000] {spark_submit.py:571} INFO - 24/01/29 09:24:21 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:24:21.126+0000] {spark_submit.py:571} INFO - 24/01/29 09:24:21 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 4 failover attempts. Trying to failover after sleeping for 30537ms.
[2024-01-29T09:24:52.665+0000] {spark_submit.py:571} INFO - 24/01/29 09:24:52 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:24:53.666+0000] {spark_submit.py:571} INFO - 24/01/29 09:24:53 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:24:54.667+0000] {spark_submit.py:571} INFO - 24/01/29 09:24:54 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:24:55.668+0000] {spark_submit.py:571} INFO - 24/01/29 09:24:55 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:24:56.669+0000] {spark_submit.py:571} INFO - 24/01/29 09:24:56 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:24:57.670+0000] {spark_submit.py:571} INFO - 24/01/29 09:24:57 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:24:58.671+0000] {spark_submit.py:571} INFO - 24/01/29 09:24:58 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:24:59.672+0000] {spark_submit.py:571} INFO - 24/01/29 09:24:59 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:25:00.673+0000] {spark_submit.py:571} INFO - 24/01/29 09:25:00 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:25:01.673+0000] {spark_submit.py:571} INFO - 24/01/29 09:25:01 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:25:01.678+0000] {spark_submit.py:571} INFO - 24/01/29 09:25:01 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 5 failover attempts. Trying to failover after sleeping for 29782ms.
[2024-01-29T09:25:32.462+0000] {spark_submit.py:571} INFO - 24/01/29 09:25:32 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:25:33.463+0000] {spark_submit.py:571} INFO - 24/01/29 09:25:33 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:25:34.463+0000] {spark_submit.py:571} INFO - 24/01/29 09:25:34 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:25:35.464+0000] {spark_submit.py:571} INFO - 24/01/29 09:25:35 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:25:36.465+0000] {spark_submit.py:571} INFO - 24/01/29 09:25:36 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:25:37.466+0000] {spark_submit.py:571} INFO - 24/01/29 09:25:37 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:25:38.467+0000] {spark_submit.py:571} INFO - 24/01/29 09:25:38 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:25:39.468+0000] {spark_submit.py:571} INFO - 24/01/29 09:25:39 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:25:40.469+0000] {spark_submit.py:571} INFO - 24/01/29 09:25:40 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:25:41.470+0000] {spark_submit.py:571} INFO - 24/01/29 09:25:41 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:25:41.478+0000] {spark_submit.py:571} INFO - 24/01/29 09:25:41 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 6 failover attempts. Trying to failover after sleeping for 29563ms.
[2024-01-29T09:26:12.043+0000] {spark_submit.py:571} INFO - 24/01/29 09:26:12 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:26:13.044+0000] {spark_submit.py:571} INFO - 24/01/29 09:26:13 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:26:14.045+0000] {spark_submit.py:571} INFO - 24/01/29 09:26:14 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:26:15.045+0000] {spark_submit.py:571} INFO - 24/01/29 09:26:15 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:26:16.046+0000] {spark_submit.py:571} INFO - 24/01/29 09:26:16 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:26:17.048+0000] {spark_submit.py:571} INFO - 24/01/29 09:26:17 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:26:18.049+0000] {spark_submit.py:571} INFO - 24/01/29 09:26:18 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:26:19.050+0000] {spark_submit.py:571} INFO - 24/01/29 09:26:19 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:26:20.053+0000] {spark_submit.py:571} INFO - 24/01/29 09:26:20 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:26:21.056+0000] {spark_submit.py:571} INFO - 24/01/29 09:26:21 INFO Client: Retrying connect to server: 0.0.0.0/0.0.0.0:8032. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1000 MILLISECONDS)
[2024-01-29T09:26:21.061+0000] {spark_submit.py:571} INFO - 24/01/29 09:26:21 INFO RetryInvocationHandler: java.net.ConnectException: Your endpoint configuration is wrong; For more details see:  http://wiki.apache.org/hadoop/UnsetHostnameOrPort, while invoking ApplicationClientProtocolPBClientImpl.getNewApplication over null after 7 failover attempts. Trying to failover after sleeping for 43970ms.
[2024-01-29T09:26:21.190+0000] {process_utils.py:131} INFO - Sending Signals.SIGTERM to group 80841. PIDs of all processes in the group: [80842, 80841]
[2024-01-29T09:26:21.208+0000] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 80841
[2024-01-29T09:26:21.209+0000] {taskinstance.py:2450} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-01-29T09:26:21.210+0000] {spark_submit.py:697} INFO - Sending kill signal to spark-submit
[2024-01-29T09:26:23.864+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/juniortemgoua0/DataLake/venv/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/juniortemgoua0/DataLake/venv/lib/python3.8/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self._application)
  File "/home/juniortemgoua0/DataLake/venv/lib/python3.8/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 490, in submit
    self._process_spark_submit_log(iter(self._submit_sp.stdout))  # type: ignore
  File "/home/juniortemgoua0/DataLake/venv/lib/python3.8/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 539, in _process_spark_submit_log
    for line in itr:
  File "/home/juniortemgoua0/DataLake/venv/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2452, in signal_handler
    raise AirflowException("Task received SIGTERM signal")
airflow.exceptions.AirflowException: Task received SIGTERM signal
[2024-01-29T09:26:23.877+0000] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=manage_data_lake_dag, task_id=spark_job_formatted, execution_date=20240129T092150, start_date=20240129T092200, end_date=20240129T092623
[2024-01-29T09:26:23.937+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 75 for task spark_job_formatted (Task received SIGTERM signal; 80841)
